src/js/profile/chart.js

```diff
@@ -237,7 +240,59 @@ export class Chart extends Observable {
       ],
     };
 
-    embed(this.container, barChart, { actions: false})
+    const calculatePosition = (event, tooltipBox, offsetX, offsetY) => {
+      let x = event.pageX + offsetX;
+      if (x + tooltipBox.width > window.innerWidth) {
+        x =+ event.pageX - offsetX - tooltipBox.width;
+      }
+      let y = event.pageY + offsetY;
+      if (y + tooltipBox.height > window.innerHeight) {
+        y =+ event.pageY - offsetY - tooltipBox.height;
+      }
+      return {x, y};
+    }
+
+    const handler = (_, event, item, value) => {
+      const tooltipClassSubstr = tooltipClass.substring(1)
+      this.el = $(tooltipClass) ? $(tooltipClass)[0] : null;
+      if (!this.el) {
+        this.el = document.createElement('div');
+        this.el.classList.add(tooltipClassSubstr);
+        document.body.appendChild(this.el);
+      }
+      const tooltipContainer = document.fullscreenElement != null ? document.fullscreenElement : document.body;
+      tooltipContainer.appendChild(this.el);
+      // hide tooltip for null, undefined, or empty string values
+      if (value == null || value === '') {
+        this.el.remove()
+        return;
+      }
+      // set the tooltip content
+      this.el.innerHTML = `
+        <div class="bar-chart__row_tooltip-card">
+          <div class="bar-chart__tooltip_name">
+              <div>${value.group}</div>
+          </div>
+          <div class="bar-chart__tooltip_value">
+              <div>${value.percentage}%</div>
```

Comment:
you can drop the % here. it's done via the format.

Reply:
removed, this was from an older iteration